packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -294,6 +295,43 @@ export class CfnParser {
     return cfnValue;
   }
 
+  public handleAttributes(resource: CfnResource, resourceAttributes: any, logicalId: string): void {
+
+    const finder = this.options.finder;
+    const cfnParser = new CfnParser({
+      finder,
+    });
+
+    const cfnOptions = resource.cfnOptions;
+
+    cfnOptions.creationPolicy = cfnParser.parseCreationPolicy(resourceAttributes.CreationPolicy);
+    cfnOptions.updatePolicy = cfnParser.parseUpdatePolicy(resourceAttributes.UpdatePolicy);
+    cfnOptions.deletionPolicy = cfnParser.parseDeletionPolicy(resourceAttributes.DeletionPolicy);
+    cfnOptions.updateReplacePolicy = cfnParser.parseDeletionPolicy(resourceAttributes.UpdateReplacePolicy);
```

Comment:
I believe `parseCreationPolicy`, `parseUpdatePolicy` and `parseDeletionPolicy` can now be made private in `CfnParser`.

Reply:
`parseUpdateReplacePolicy` as well?